### PR TITLE
Bounty Retool

### PR DESCRIPTION
**Pull request by Kaedwuff**
The cargo bounty system appears to have been lifted wholesale from another server with much lower standards.  It's full of nonsense and memes and the rewards are all astronomically high compared to the economic value of items in cargo, which is the currently only benchmark we have for the value of items on the server other than merchants.  In some cases, it was possible to make huge profits simply ordering an item in cargo and sending it to the ODIN.  This is not balanced nor logical.

As such, I have overhauled the bounty system. Summary of changes:

-All values of bounties have been drastically reduced for sanity purposes. For the most part, all of them have been either reduced to 10%-50% of their previous value or retooled to reward a small amount based on the number/rarity of items in the bounty. Some more difficult to obtain items have higher values, but nothing like the old ones.  No longer can you cut up a few carrots grown in hydroponics and make over half the value of the cargo account instantly.  Items that can be ordered from cargo will always net you a loss in value, so you can't just game the system using the shipping.
-Anything that was an antagonist or illegal item has been removed.  Items in the bounty list should be possible to obtain by working with crew members, and not require a certain round type antag to be possible. Some items that encouraged illegal/questionable activity, like 'soylent green' or 'corgi meat' are also off the list now.  The ODIN is run by humans who are not insane.  They are not going to ask for human or dog meat, cyanide or night vision goggles.
-All items that are unique and belong to command staff/security have been removed, due to a tendency for cargo to waste large amounts of time badgering them to give up things like their captain's spare or telebatons.  These items should not be handed to cargo, nor should we be giving the cargo team a reason to feel they're obligated to it.
-Any bounty that was excessively silly or memey (talk of cow riots or tajaran uprisings, etc) has either been redescribed to stop being so, or has been removed if it is unsalvagable.

https://forums.aurorastation.org/topic/11849-cargo-retool/

### CODE REVIEW

**Comment by Kaedwuff** [comment]
!review

File: code/modules/cargo/bounties/assistant.dm [hunk 1]
```diff
@@ -49,127 +49,88 @@
...
+	description = "Commander Pat is unhappy with his chair. He claims it hurts his back. Ship some alternatives out, please. "
+	reward = 200
 	required_count = 5
 	wanted_types = list(/obj/structure/bed/chair/comfy)
 
 /datum/bounty/item/assistant/revolver
 	name = "Revolver"
 	description = "One of our Lead Investigators lost their revolver. He's asked for help securing an appropriate replacement."
-	reward = 2000
+	reward = 750
```
Comment (Sindorman) [inline code/modules/cargo/bounties/assistant.dm]: potted plant is 800. But a gun is 750?

File: code/modules/cargo/bounties/assistant.dm [hunk 2]
```diff
@@ -183,16 +144,4 @@
 		return T.air_contents.gas["phoron"] >= moles_required
 	return FALSE
 
-/datum/bounty/item/assistant/corgimeat
-	name = "Raw Corgi Meat"
-	description = "The Syndicate recently stole all of %BOSSSHORT's corgi meat. Ship out a replacement immediately."
-	reward = 3000
-	wanted_types = list(/obj/item/weapon/reagent_containers/food/snacks/meat/corgi)
-
-/datum/bounty/item/chef/action_figures
```
Comment (Sindorman) [inline code/modules/cargo/bounties/assistant.dm]: Put it back, but reduce the price. Notice it is 5 of them

File: code/modules/cargo/bounties/chef.dm [hunk 3]
```diff
@@ -1,134 +1,106 @@
...
 	wanted_types = list(/obj/item/weapon/reagent_containers/food/snacks/muffin)
 
 /datum/bounty/item/chef/chawanmush
 	name = "Chawanmushi"
 	description = "Someone from middle-management mentioned Chawanmushi. We never tried them so we need you to ship Chawanmushi immediately."
-	reward = 8000
+	reward = 300
 	wanted_types = list(/obj/item/weapon/reagent_containers/food/snacks/chawanmushi)
 
-/datum/bounty/item/chef/kebab
```
Comment (Sindorman) [inline code/modules/cargo/bounties/chef.dm]: why did you remove this?

File: code/modules/cargo/bounties/mech.dm [hunk 4]
```diff
@@ -15,22 +15,22 @@
...
 /datum/bounty/item/mech/odysseus
 	name = "Odysseus"
-	reward = 11000
+	reward = 2700
 	wanted_types = list(/obj/mecha/medical/odysseus)
 
 /datum/bounty/item/mech/gygax
 	name = "Gygax"
-	reward = 28000
+	reward = 4500
```
Comment (Sindorman) [inline code/modules/cargo/bounties/mech.dm]: according to ED 209 prices, this is 4.5 of ED 209. Yet making ED 209 is extremely easy and would not be of much of cost of materials. Gygax on the other hand uses diamonds and lots of materials plus research components, etc. And Labor too. This is underpriced

File: code/modules/cargo/bounties/mech.dm [hunk 5]
```diff
@@ -15,22 +15,22 @@
...
 	wanted_types = list(/obj/mecha/combat/gygax)
 
 /datum/bounty/item/mech/durand
 	name = "Durand"
-	reward = 20000
+	reward = 4000
```
Comment (Sindorman) [inline code/modules/cargo/bounties/mech.dm]: same as my comment above

File: code/modules/cargo/bounties/mech.dm (hunk 4 above)
Comment (Sindorman) [reply code/modules/cargo/bounties/mech.dm]: PS. Diamond drill cost 2000, which means that just diamonds along would be 2000 in creating Gygax

File: code/modules/cargo/bounties/security.dm [hunk 6]
```diff
@@ -1,62 +1,37 @@
...
+	reward = 500
 	wanted_types = list(/obj/item/weapon/pinpointer)
 
-/datum/bounty/item/security/captains_spare
-	name = "Captain's Spare"
-	description = "Captain Bart of Station 12 has forgotten his ID! Ship him your station's spare, would you?"
-	reward = 1500
-	wanted_types = list(/obj/item/weapon/card/id/captains_spare)
-
-/datum/bounty/item/security/hardsuit
```
Comment (Sindorman) [inline code/modules/cargo/bounties/security.dm]: You should replace this with RIG imho. Since RIGs can be made.

**Comment by skull132** [comment]
Unaddressed changes from over 2 months ago. Time to nook.
